### DISCUSSION TITLE
Group Meeting Notes

### DISCUSSION THREAD

**Discussion by alexanderfordetamu** [alexanderfordetamu/Introducing-Group-5#7] [General]
Discuss progress here!

**Comment by alexanderfordetamu** [comment]
Great merge @delbmatt

**Comment by zkeck** [comment]
Start creating your own content.

**Comment by zkeck** [comment]
Found that the website does not immediately update.

**Comment by delbmatt** [comment]
The website seems to locally cashe so you have to hard refresh in order to see changes

**Comment by alexanderfordetamu** [comment]
@mram6525 slacked our group about your pull request.

**Comment by mram6525** [comment]
Just sent pull request fixing Kelly's bio.

**Comment by mram6525** [comment]
Done
